conf/db/migration/metrics_portal_ddl/common/V11__create_reporting_tables.sql

```diff
@@ -0,0 +1,73 @@
+-- Copyright 2018 Dropbox, Inc.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+CREATE TABLE portal.report_sources (
+    id BIGSERIAL PRIMARY KEY,
+    uuid UUID NOT NULL,
+    version INTEGER NOT NULL DEFAULT 1,
+    created_at TIMESTAMP NOT NULL DEFAULT now(),
+    updated_at TIMESTAMP NOT NULL DEFAULT now(),
+    timeout_seconds INTEGER NOT NULL,
+    type VARCHAR(255) NOT NULL,
+
+    -- Chrome
+    url VARCHAR(255),
+    title VARCHAR(255) DEFAULT '',
+    ignore_certificate_errors BIT DEFAULT 0,
+    triggering_event_name VARCHAR(255),
+    pdf_width_inches FLOAT,
+    pdf_height_inches FLOAT,
+);
+
+CREATE UNIQUE INDEX report_sources_uuid_idx ON portal.report_sources (uuid);
+
+CREATE TABLE portal.report_recipient_groups (
+    id BIGSERIAL PRIMARY KEY,
+    uuid UUID NOT NULL,
+    version INTEGER NOT NULL DEFAULT 1,
+    name VARCHAR(255) DEFAULT '',
```

Comment:
So is the idea to have full on group management? Or does each report create its own "group" (just wondering what the role of name is)

Reply:
The name might not be precise enough, I wanted to encapsulate who the report is sent to, rather than storing it as a "Sink" since that concept only makes sense to me at runtime. 

Really all this is trying to be is a set of recipients for a single report, with all of the individual addresses in an umbrella underneath it.